.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -24,20 +24,20 @@ assignees: robertsLando
 
   4. Make sure to provide the necessary information, as described here:
   https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/bug_report
-  
+
   If you are using HomeAssistant, this is how you do it:
   Home Assistant -> settings -> Integrations -> Z-Wave JS -> Configure -> Create dump -> zip the json file and post it here.
 
-  🙏🏻🙏🏻🙏🏻 Thanks, now onto your issue:
+  🙏🏻🙏🏻🙏🏻 Thanks, now on to your issue:
 -->
 
 ### Version
 
 **Checklist:**
 
```

Comment:
I think this is a good chance to convert this into an issue form like in node-zwave-js
==> https://issue-forms-creator.app/new helps with that
==> For an example: https://github.com/zwave-js/node-zwave-js/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml

Reply:
Also, should we just require application and driver logs here?

Reply:
@blhoward2 I would keep this for another PR if you want